crates/object_detection/src/pose_detection.rs

```diff
@@ -222,6 +222,36 @@ impl PoseDetection {
             human_poses: poses.into(),
         })
     }
+
+    fn should_look_for_referee<T: TimeInterface>(
+        &mut self,
+        hardware_interface: &Arc<T>,
+        motion_command: &MotionCommand,
+        pose_detection_inactive_duration: &Duration,
+    ) -> bool {
+        let is_in_initial_with_pose_detection = matches!(
+            motion_command,
+            MotionCommand::Initial {
+                should_look_for_referee: true,
+                ..
+            }
+        );
+
+        if is_in_initial_with_pose_detection && !self.pose_detection_was_started {
+            self.start_time = hardware_interface.get_now();
+            self.pose_detection_was_started = true;
+        } else if !is_in_initial_with_pose_detection {
+            self.start_time = hardware_interface.get_now();
+        }
+
+        let inactive_period_as_elapsed = hardware_interface
+            .get_now()
+            .duration_since(self.start_time)
+            .ok()
+            > Some(*pose_detection_inactive_duration);
```

Comment:
huh?! this is possible? You are ordering options here? What happens if it is `None`?

Reply:
It is. A `None` is always smaller than any other value. But is also removed, as mentioned above.